### DISCUSSION TITLE
Setting 'busy' for event titles

### DISCUSSION THREAD

**Discussion by peckevin-github** [derekantrican/GAS-ICS-Sync#370] [Q&A] [answered]
Is it possible to modify the sync so that the title is simply 'busy' for anything that's been synced? I'd love it as well if I could remove the location and description of the event so that it's simply marked as a busy time for me rather than having the event details added to the Google calendar.

thanks!

**Comment by jonas0b1011001** [comment, marked as answer]
Sure. The event title is set in [Helpers.gs line 360](https://github.com/derekantrican/GAS-ICS-Sync/blob/e5cd8e02417ee38a04eadbd795201822dbac2e2d/Helpers.gs#L360), just change it to
`newEvent.summary = 'busy';`

Just look through [function createEvent()](https://github.com/derekantrican/GAS-ICS-Sync/blob/e5cd8e02417ee38a04eadbd795201822dbac2e2d/Helpers.gs#L263) and you will find every property, just remove the respective lines for each property you want to ignore, e.g.:
description, [Helpers.gs lines 383+384](https://github.com/derekantrican/GAS-ICS-Sync/blob/e5cd8e02417ee38a04eadbd795201822dbac2e2d/Helpers.gs#L383)
```
  if (event.hasProperty('description'))
    newEvent.description = icalEvent.description;
```
location, [Helpers.gs lines 386+387](https://github.com/derekantrican/GAS-ICS-Sync/blob/e5cd8e02417ee38a04eadbd795201822dbac2e2d/Helpers.gs#L386)
```
  if (event.hasProperty('location'))
    newEvent.location = icalEvent.location;
```

**Reply by peckevin-github** [reply]
Amazing, thank you!
